### ISSUE TITLE
Notification email digest

### ISSUE THREAD

**Issue by BNAndras** [exercism/exercism#6578]
Would it be possible to have a hourly or daily digest email option for mentoring notifications? I've had several students submit multiple iterations and comments in short succession, and each one gets an email.

**Comment by iHiD** [comment]
Yeah - it's something that's been on the radar for ages but never gets to the top of the priority list. Thanks for flagging it up.
